Put more details about in the Java section (#1525)
platformsh/platformsh-docs: docs/src/languages/java/_index.md

## Other JVM languages

> It’s worth remembering that the JVM by its specification [does not read Java code](https://docs.oracle.com/javase/specs/jvms/se8/html/index.html), but bytecode. So within the JVM, it’s possible to [run several languages](https://en.wikipedia.org/wiki/List_of_JVM_languages). Because the JVM is polyglot, Platform.sh has support to several such as Kotlin, Groovy, Scala, and so on. As long as that language works with any build automation that Platform.sh supports.

Reviewer: ```suggestion
It’s worth remembering that the JVM by its specification [does not read Java code](https://docs.oracle.com/javase/specs/jvms/se8/html/index.html), but bytecode. So within the JVM, it’s possible to [run several languages](https://en.wikipedia.org/wiki/List_of_JVM_languages). Platform.sh supports several of them, such as Kotlin, Groovy, and Scala, so long as that language works with any build automation that Platform.sh supports.
```

{{< /codetabs >}}

 {{< note >}}

> Those samples are using the [Configuration Reader](https://github.com/platformsh/config-reader-java) that Platform.sh provides to you if you wish, however, if your favorite Java frameworks allow you to overwrite the configurations following [The Twelve-Factor App](https://12factor.net/config) such as Spring, Micronaut, Quarkus, Jakarta EE/MicroProfile, and so on. 

Reviewer: ```suggestion

Those samples are using the [Configuration Reader](https://github.com/platformsh/config-reader-java) that Platform.sh provides to you if you wish, however, if your favorite Java frameworks allow you to overwrite the configurations following [The Twelve-Factor App](https://12factor.net/config) such as Spring, Micronaut, Quarkus, Jakarta EE/MicroProfile, and so on. 

```

Reviewer: The  sentence  seems like its missing an ending: "(use provided config  reader), however (if  frameworks allow overwrite along 12FA like A,B, C),  then ...

Author: Thank you, I'll accept your suggestion and then refactor it.

> {{< /note }}

Reviewer: ```suggestion
{{< /note >}}
```

Reviewer: ```
    Building sites … Total in 6752 ms
    W: Error: Error building site: "/app/src/languages/java/_index.md:123:11": unclosed shortcode
```

> Those samples are using the [Configuration Reader](https://github.com/platformsh/config-reader-java) that Platform.sh provides to you if you wish, however, if your favorite Java frameworks allow you to overwrite the configurations following [The Twelve-Factor App](https://12factor.net/config) such as Spring, Micronaut, Quarkus, Jakarta EE/MicroProfile, and so on. Then, as a developer you have the option to don't use the configure library and don't impact your code with any third dependency.

Reviewer: ```suggestion
These samples use the [Configuration Reader library](https://github.com/platformsh/config-reader-java) that Platform.sh provides. However, if your favorite Java frameworks allow you to overwrite the configurations following [The Twelve-Factor App](https://12factor.net/config) (such as Spring, Micronaut, Quarkus, Jakarta EE/MicroProfile), then as a developer you have the option to not use the library, removing the need for an additional dependency.
```
